pkgs/development/compilers/rust/default.nix

```diff
@@ -25,11 +25,6 @@ in rec {
     # So we do the same.
     # 2. Tests run out of memory for i686
     doCheck = !stdenv.isAarch64 && !stdenv.isi686;
-
-    patches = [
```

Comment:
patches is a required argument

Reply:
Ah, right. I'll fix this tomorrow. Should I have kept the `grsec.patch` thing by the way? I have no idea what that is.

Reply:
@jD91mZM2 we have removed grsec support in nixos since 17.09. You can drop the patch.